specification/keyvault/data-plane/Microsoft.KeyVault/2016-10-01/keyvault.json

```diff
@@ -1164,7 +1169,7 @@
         ],
         "responses": {
           "200": {
-            "description": "A response message containing a list of deleted secrets in the vault along with a link to the next page of deleted secrets",
+            "description": "A response message containing a list of deleted secrets in the vault, along with a link to the next page of deleted secrets.",
```

Comment:
Since you're updating the descriptions, it would be great to be consistent about including the required permission. I don't have a preference (including them or not), just a consistency (minor) issue.

Reply:
I think the included permissions being explicit on the operations is good? Otherwise each response would also need an additional "required permissions" which seems like overkill.